composer/callbacks/mlperf.py

```diff
@@ -320,8 +320,8 @@ def epoch_start(self, state: State, logger: Logger) -> None:
 
     def epoch_end(self, state: State, logger: Logger) -> None:
         if _global_rank_zero():
-            self.mllogger.event(key=constants.EPOCH_STOP, metadata={'epoch_num': self._get_time(state)})
-            logger.file_artifact(artifact_name=self.upload_name, file_path=self.filename)
+            self.mllogger.event(key=constants.EPOCH_STOP, metadata={'epoch_num': state.timestamp.epoch.value})
```

Comment:
I think you probably didn't mean to change this back to `state.timestamp.epoch.value` in the merge?

Reply:
ah, good catch, thank you!